index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="Hello Microverse">
+    <meta name="keywords" content="HTML, CSS, Linter">
+    <meta name="author" content="Oussedit">
+    <link rel="stylesheet" href="./style.css">
+    <title>Portfolio</title>
+</head>
+
+<body>
+    <div class="wrapper">
+        <header>
+            <a href="">
+                <div id="logo">My logo</div>
+            </a>
+            <nav>
+                <div><img src="./assets/ham.svg" alt="" > </div>
+            </nav>
+        </header>
+    </div>
+    <main>
+        <section id="sec1">
+
+            <div>
+                <h1 id="nm">I’m Mariane<br><br>
+                    Glad to see<br><br> you!</h1>
+
+                <p class="desc">I’m a software developer! I can help you build a product , feature or website Look
+                    through
+                    some of my work and experience! If you like what you see and have a project you need coded, don’t
+                    hestiate to contact me.</p>
+
+                <a href="#" class="lnk">LET’S CONNECT</a>
+
+
+                <div class="socials">
+                    <ul>
+                        <img src="./assets/Vector.svg" alt="">
+                        <li><a href=""></a></li>
+                        <img src="./assets/Vector (1).svg" alt="">
+                        <li><a href=""></a></li>
+                        <img src="./assets/Vector (2).svg" alt="">
+                        <li><a href=""></a></li>
+                        <img src="./assets/Vector (3).svg" alt="">
+                        <li><a href=""></a></li>
```

Comment:
- [ ]  the previous reviewer meant to wrap each `<img>` tag with `<a>` tag.
so it would look like this:

``` 
<ul>
  <li> 
     <a> <img> </a>
  </li>
</ul>
```
I would suggest you to wrap <img> tags with <a> tags.